target_chains/solana/sdk/js/pyth_solana_receiver/examples/post_twap_update.ts

```diff
@@ -0,0 +1,94 @@
+import { Connection, Keypair, PublicKey } from "@solana/web3.js";
+import { InstructionWithEphemeralSigners, PythSolanaReceiver } from "../";
+import { Wallet } from "@coral-xyz/anchor";
+import fs from "fs";
+import os from "os";
```

Comment:
Where you able to test this example works?

Reply:
working on it, will update when i've validated it works e2e

Reply:
it works! can check out the txs from the output: https://app.warp.dev/block/dS1PbuFr3e4CgtG4MXMfCq